### DISCUSSION TITLE
Resuming GitHub app (previously on notifications inbox) triggers automatic refresh. But the list of notifications is now truncated

### DISCUSSION THREAD

**Discussion by jasonkarns** [community/community#39034] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.88.0 (82868297)
Device: iPad 6 (iOS 16.1)
</details>

Force refreshing (pull to refresh or killing/restarting app doesn't cause them to reappear.

**Comment by aconsuegra** [comment]
Hi @jasonkarns, thanks for getting in touch! Do you mind including a screenshot or recording of the steps to reproduce the issue and how the list of notifications look for you?
